src/app/components/ConsentBanner/index.stories.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import {
+  ConsentBanner,
+  ConsentBannerButton,
+  ConsentBannerLink,
+  ConsentBannerText,
+} from '.';
+
+const Accept = <ConsentBannerButton onClick={() => {}}>OK</ConsentBannerButton>;
+
+const Reject = (
+  <ConsentBannerLink href="https://www.bbc.co.uk/usingthebbc/your-data-matters">
+    Find out what&apos;s changed
+  </ConsentBannerLink>
+);
+
+const Text = (
+  <ConsentBannerText>
+    We have made some <a href="https://www.bbc.com/news">important</a> changes
```

Comment:
This link is weird 🤔

Reply:
![image](https://user-images.githubusercontent.com/11341355/54299968-e1b97100-45b3-11e9-81ff-9d337a8f08ec.png)

Its to simulate the real banner, Ive updated the text to make it clearer